cmd/e2e-runner/main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 }
 
 func randomString() string {
-	rand.Seed(time.Now().Unix())
+	rand.Seed(time.Now().UnixNano())
```

Comment:
please us crypto/rand instead of rand

Reply:
it's for consistency w/ the rest of the codebase, we're always using crypto/rand instead of rand package

Reply:
Yup, change math/rand to crypto/rand in the imports and remove this line altogether.